public/javascripts/filters_old.min.js

```diff
@@ -0,0 +1 @@
+function showFilters(){var e=$j("dd.tags");e.each(function(e,r){var t=$j(r).find("input"),i=$j(r),n=$j("."+i.attr("id")+"_open"),s=$j("."+i.attr("id")+"_close");t.each(function(e,t){$j(t).is(":checked")&&($j(r).show(),$j(n).hide(),$j(s).show())})})}function setupNarrowScreenFilters(){var e=$j("form.old-filters"),r=$j("#outer"),t=$j("#go_to_filters"),i=$j("#leave_filters");t.click(function(t){t.preventDefault(),e.removeClass("narrow-hidden"),r.addClass("filtering"),e.find(":focusable").first().focus(),e.trap()}),i.click(function(i){i.preventDefault(),r.removeClass("filtering"),e.addClass("narrow-hidden"),t.focus()})}$j(document).ready(function(){showFilters(),setupNarrowScreenFilters()});
```

Comment:
I'm only restoring the minified version of the old filter JavaScript because I couldn't think of a good reason to restore the non-minified version.